frame/balances/src/tests_local.rs

```diff
@@ -76,7 +76,7 @@ impl frame_system::Config for Test {
 	type PalletInfo = ();
 	type AccountData = super::AccountData<u64>;
```

Comment:
it would make more sense to have it set to nothing when balance is stored locally.
Anyway it should be unused.
```suggestion
	type AccountData = ();
```